packages/@atjson/document/test/document-test.ts

```diff
@@ -0,0 +1,140 @@
+import TestSource, { Bold, Paragraph } from './test-source';
+import { ParseAnnotation } from '../src';
+
+describe('Document#canonical', () => {
+  test('parse tokens are properly removed', () => {
+    let testDoc = new TestSource({
+      content: '<b>Hello</b>,\n World!',
+      annotations: [
+        new ParseAnnotation({ start: 0, end: 3 }),
+        new ParseAnnotation({ start: 8, end: 12 }),
+        new Paragraph({ start: 0, end: 13 }),
+        new Bold({ start: 0, end: 12 })
+      ]
+    });
+    expect(testDoc.canonical()).toMatchObject({
+      contentType: 'application/vnd.atjson+test',
+      content: 'Hello,\n World!',
+      annotations: [{
+        type: 'bold',
+        start: 0,
+        end: 5,
+        attributes: {}
+      }, {
+        type: 'paragraph',
+        start: 0,
+        end: 6,
+        attributes: {}
+      }]
+    });
+  });
+
+  test.only('documents are correctly compared for equality', () => {
```

Comment:
```suggestion
  test('documents are correctly compared for equality', () => {
```

You're only running a single test here.